### ISSUE TITLE
Improve git commit comments

### ISSUE THREAD

**Issue by Qubitium** [qwopqwop200/GPTQ-for-LLaMa#117]
@qwopqwop200  You have been committing at a rate that is exceeding human expectations. Unfortunately, this leads to extreme variance from hour-to-hour and day-to-day as only two (technically one now that triton is getting most the attention) branch of code.  There are growing user related issues as far as compat and performance divergence that is starting to appear. 

1. Please label commits into relevant categories; i.e  FIX, REFRACTOR, ADD, CLEAN, etc. It doesn't matter the format or how they are labeled. We just need them correctly labeled to better track changes, without having to peel into code every time. 
2. Perhaps starting a "dev" branch that is only merged to triton (master/main) after user feedbacks or further testing? 

We are extremely appreciative of your work and only want to help this repo improve going forward as the speed of development is quite amazing.  Thank you!

**Comment by qwopqwop200** [comment]
1. I see. From this point on, I will add comments related to commits.
2. From now on, we will add a dev branch.

I think the current version is the stable version. Currently, we have implemented all the additional features of GPTQ, and the kernel also shows very impressive speed compared to CUDA. We also implemented backpropagation.

Currently, the next feature I'm thinking of is the implementation of FLAN-T5, and if possible, I will also implement GALACTICA.
